### PR TITLE
Fix names

### PR DESCRIPTION
**Pull request by redferret**
Fixed names on hash in method merchants_as_mocks
Added attributes `updated_at` and `created_at` to Merchant class

Removed unrecognized Cop (not sure what it's not recognized anymore)
- [x] Tests are passing

### CODE REVIEW

File: data/mock_data.rb [hunk 1]
```diff
@@ -21,10 +21,10 @@ def self.merchants_as_mocks(merchant_hashes)
       raise 'Bind self of ExampleGroup to your mocks. use {self}' if not block_given?
       eg = yield
       merchant_mock = eg.instance_double('Merchant',
-        name: item_hash[:name],
-        id: item_hash[:id],
-        created_at: item_hash[:created_at],
-        updated_at: item_hash[:updated_at]
+        name: merchant_hash[:name],
+        id: merchant_hash[:id],
+        created_at: merchant_hash[:created_at],
```
Comment (mkrumholz) [inline data/mock_data.rb:26]: Merchant class doesn't have created_at/updated_at attributes, so this may still cause issues.

Comment (mkrumholz) [reply data/mock_data.rb:26]: We may need to add these as attributes to the merchant class if we need them, or remove them from these mocks if we do not.

File: lib/merchant.rb [hunk 2]
```diff
@@ -7,6 +7,8 @@ class Merchant
   def initialize(details)
     @id = details[:id]
     @name = details[:name]
+    @created_at = details[:created_at]
```
Comment (mkrumholz) [inline lib/merchant.rb:12]: Nice! Thanks for adding these -- looks like they will be useful in Iteration 4

**Review by mkrumholz — APPROVED** [review]
Looks good now with the changes to Merchant class -- we may still need some review on the spec here.

**Comment by redferret** [comment]
> Some files could not be reviewed due to errors:
> 
> Warning: unrecognized cop Lint/AmbiguousAssignment found in .rubocop.yml

I don't know why this is happening, it seems to work fine when running `rubocop -A` Each time I remove these it seems to complain about the next one. So I'll just leave it alone for now

**Review by NoahZinter — APPROVED** [review]
Looks like we have merchant.rb on the same page with mock_data.rb now

**Pull request merged**